### PR TITLE
Add is staff transform

### PR DESCRIPTION
**Pull request by tom-parker**
cc @commuterjoy @adambraimbridge

### CODE REVIEW

**Comment by adambraimbridge** [comment]
:+1:

File: server/transforms/isStaff.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var metrics = require('next-metrics');
+
+module.exports = function (event) {
+	return new Promise((resolve, reject) => {
+		if (!process.env.transform_isStaff && !process.env.mocha) {
```
Comment (commuterjoy) [inline server/transforms/isStaff.js]: for some reason I've kept all the env vars to lower case underscore separated.

File: server/transforms/isStaff.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+			console.log('transforms/isStaff', 'is switched off');
+			return resolve({});
+		}
+
+		metrics.count('pipeline.transforms.isStaff.count', 1);
```
Comment (commuterjoy) [inline server/transforms/isStaff.js]: same here. lose the casing.

File: server/transforms/isStaff.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+		var cookie = event.headers().cookie || '';
+		var isStaff = cookie.match(/FT_User=[^;]+(\@ft\.com)/i);
```
Comment (commuterjoy) [inline server/transforms/isStaff.js]: I always write these as one liners ...

`var isStaff = /FT_User=[^;]+(\@ft\.com)/i.test(event.headers().cookie);`

**Comment by commuterjoy** [comment]
:+1: good to go apart from my comments

**Pull request merged**